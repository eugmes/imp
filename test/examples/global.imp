var g : integer;
-- Unused variable for testing
var x : boolean;

procedure main () is
begin
  output("Enter number: ");
  input(g);
  output("Number is: "); output(g); newline;
end main;
