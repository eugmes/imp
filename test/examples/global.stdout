Enter number: Number is: 42
